packages/react/src/ReactRenderer.tsx

```diff
@@ -116,12 +107,6 @@ export class ReactRenderer<R = unknown, P = unknown> {
       this.element.classList.add(...className.split(' '))
     }
 
-    if (attrs) {
-      Object.keys(attrs).forEach(key => {
-        this.element.setAttribute(key, attrs[key])
-      })
-    }
-
```

Comment:
I would prefer to keep this updating of the element within the ReactRenderer rather than have the `ReactNodeViewRenderer` managing it.

Could we add a method here that basically does this code here, named `updateAttributes` which `ReactNodeViewRenderer` calls when it wants to apply the changed attributes.